docs-chef-io/content/server/ctl_chef_server.md

```diff
@@ -955,6 +955,17 @@ This subcommand has the following options:
     option only works when run on a standalone Chef Infra Server or on a
     primary backend Chef server within a legacy tier.
 
+{{< note >}}
+If the search does not work when data is present even after reindex, please verify the configurations of the search index. The following command will get the configuration of search index :
```

Comment:
s/ If knife search does not seem to return the expected results and data is present in the chef-server database after reindex.

Perhaps also add an example of what missing looks like.

Reply:
```suggestion
If `knife search` does not return the expected results and data is present in the Chef Infra Server after reindex, then verify the search index configuration with the command:
```